images/recycler/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# This is the default deployment strategy image for OpenShift Origin. It expects a set of
+# environment variables to parameterize the deploy:
+#
+#   KUBERNETES_MASTER - the address of the OpenShift master
+#   KUBERNETES_DEPLOYMENT_ID - the deployment identifier that is running this build
+#
+# The standard name for this image is openshift/origin-deployer
+#
+FROM openshift/origin
```

Comment:
nodes don't have openshift containers pulled today (they might when people have containerized node, but that's not what is done today)

RHEL7/CentOS/Fedora seems like the better base...